### PR TITLE
Fix typos and reword

### PR DESCRIPTION
**Pull request by ifreund**
#### Summary
```SUMMARY: None```

#### Purpose of change
Fixes: #28418.

#### Additional context
Thanks @Wuzzy2

### CODE REVIEW

**Comment by Wuzzy2** [comment]
Looks like you forgot the Rowland thing.

**Comment by Night-Pryanik** [comment]
We had a discussion somewhere (don't remember the exact PR though) that autodoc should be an Autodoc, capitalized.

**Comment by ifreund** [comment]
@Wuzzy2 seems to have been fixed https://github.com/CleverRaven/Cataclysm-DDA/blob/24f4cddd3474f8d88677beff4113282f01fa5792/data/json/items/ammo/460.json#L15
A couple of the other basic spelling errors were already fixed as well.

File: data/json/items/tools.json [hunk 1]
```diff
@@ -2886,7 +2886,7 @@
     "flags": [ "FISH_GOOD" ]
   },
   {
-    "id": "flamable_arrow",
```
Comment (kevingranade) [inline data/json/items/tools.json]: These aren't user facing, and changing them without migration code will break saves.
Same for every json "id" field.

**Pull request merged**